src/main/kotlin/de/bund/digitalservice/useid/identification/TcTokenController.kt

```diff
@@ -0,0 +1,41 @@
+package de.bund.digitalservice.useid.identification
+
+import io.micrometer.core.annotation.Timed
+import io.swagger.v3.oas.annotations.Operation
+import io.swagger.v3.oas.annotations.media.Content
+import io.swagger.v3.oas.annotations.responses.ApiResponse
+import io.swagger.v3.oas.annotations.tags.Tag
+import org.springframework.http.HttpStatus
+import org.springframework.http.MediaType
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.GetMapping
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.RestController
+import java.util.*
+
+internal const val TCTOKENS_BASE_PATH = "/api/v1/tc-tokens"
+
+@RestController
+@Timed
+@Tag(name = "eID-Client")
```

Comment:
It's only one endpoint, but still, adding `RequestMapping` including the base path might make sense.